src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+public class IntakeSubsystem extends SubsystemBase {
+
+    private final TalonFX intakeMotor;
+
+    private final DigitalInput rightBeamBreak;
+    private final DigitalInput leftBeamBreak;
+
+    public IntakeSubsystem() {
+        intakeMotor = new TalonFX(0); 
+        intakeMotor.setNeutralMode(NeutralModeValue.Brake);
+        //intakeMotor.setInverted(true);
```

Comment:
You will most likely want to create a `TalonFXConfiguration` object and configure this TalonFX this way with things you want such as `NeutralMode` and `Inverted`. We did this on some of the other subsystems which are good examples (Some subsystems might be in different branches). You probably don't need an `applyConfigs()` method as you dont need to change any of the config on the fly, but maybe it's worth it to keep consistency across subsystems?